### PR TITLE
Init github actions

### PR DESCRIPTION
**Pull request by glazychev-art**
Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/networkservicemesh/integration-k8s-gke
+
+go 1.15
```
Comment (denis-tingaikin) [inline go.mod:3]: Let's use go 1.16

Comment (denis-tingaikin) [reply go.mod:3]: Previously I've updated `integration-tests` and `integration-k8s-kind` so now it is critical.

Comment (glazychev-art) [reply go.mod]: Done

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+           exit 1
+          fi
+  checkgomod:
+    name: check go.mod and go.sum
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
```
Comment (denis-tingaikin) [inline .github/workflows/ci.yaml:56]: Please use go 1.16 and in other places.

Comment (glazychev-art) [reply .github/workflows/ci.yaml]: done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**